api/graphql/schema.graphql

```diff
@@ -321,8 +321,8 @@ type UserPublicProfile {
 
 type ScreeningQuestion {
   id: ID!
-  poolAdvertisement: PoolAdvertisement! @belongsTo(relation: "pool")
```

Comment:
Yeah, this is a bit heavy-handed.  I'm questioning the usefulness of having non-nullable fields in query types when it forces type contortions.
https://talent-cloud.slack.com/archives/C0259G6KXJ6/p1681410542181349?thread_ts=1680804915.429799&cid=C0259G6KXJ6